src/index.html

```diff
@@ -1,17 +1,143 @@
-    <h1>HTML Form</h1>
+    <form action="#">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input type="text" id="surname">
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input type="text" id="name"> 
+        </div>
+        
+        <div>
+          <label for="age">How old are You?</label>
+          <input 
+            class="age"
+            type="number" 
+            value="12" 
+            id="age"
+          >
+        </div>
+        
+        <div>
+          <label for="birthDate">Full date of birth:</label>
+          <input type="date" value="mm/dd/yyyy">
+        </div>
+       
+        <div>
+          <label for="agree">I accept the term of the agreement</label>
+          <input type="checkbox">
+        </div>
+
+      </fieldset>
+      <fieldset class="form-field">
+
+        <legend>Registration</legend>
+
+        <div>
+          <label for="email">E-mail: </label>
+          <input type="email" placeholder="email@example.com">
+        </div>
+       
+        <div>
+          <label for="password">Password:</label>
+          <input type="password">
+        </div>
+      
+
+      </fieldset>
+      <fieldset class="form-field">
+
+        <legend>An interesting fact about you</legend>
+        <div>
+          <span>Do you love cats?</span>
+          <input 
+            type="radio"
+            name="radioYes" 
+            id="radioYes"
+          > 
+          <label for="radioYes">Yes</label>
+        
+          <input
+            type="radio" 
+            name="radioNo" 
+            id="radioNo"
+          >  
+          <label for="radioNo">No</label>
+          
+        </div>
+        
+        
+        <div>
+          <label for="color">What is your favorite color?</label>
+          <input type="color" id="color">
+        </div>
+      
+        <div>
+          <label for="time">What time do you go to bed?</label>
+          <input 
+            type="time" 
+            id="time" 
+            step="2"
+          >
+        </div>
+       
+        <div>
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select name="cars" 
+            id="cars" multiple>
+            <option value="1">BMW</option>
+            <option value="2">Audi</option>
+            <option value="3">Lada</option>
+          </select>
+        </div>
+        <div>
+          <label for="range">How do you rate our work?</label>
+          <input
+            type="range"
+            min="1"
+            max="100"
+            value="50"
+            class="slider"
+            id="range"
+          >
+        </div>
+       
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+        <div>
+          <label>Comments:</label>
+          <textarea name="commentfield"></textarea>
```

Comment:
```suggestion
          <textarea name="commentField"></textarea>
```
[checklist p.4](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) use camelCase